### DISCUSSION TITLE
Search Filter Combo Box - Show all values

### DISCUSSION THREAD

**Discussion by ymrvprasad** [microsoft-search/pnp-modern-search#3324] [Q&A]
Hi Team,

We are using Search Results webpart along with Search Filter webpart, both are working fine. 

However, after applying a filter values in all dropdowns are getting refreshed based on the available results. Is there any possibility to show list of all values in Filter Dropdowns even if there are no results for that filter value?

**Comment by kasperbolarsen** [comment]
Hi @ymrvprasad No, that is exactly as designed and intended. 

Otherwise, the end user will filter by one value and get a lot of additional filter options, all of them or most of them showing "0 results found" when selecting them.

**Reply by ymrvprasad** [reply]
hi @kasperbolarsen,

Sorry for the delay in responding back. 

We are OK to have Filter values based on Search Query in page load. But we don't want to get those filters refreshed after selecting any filter value. This is a mandatory requirement for one of our client. Could you please suggest a way to achieve this.

**Reply by kasperbolarsen** [reply]
Hi @ymrvprasad 
In that case your client are asking for something which are against best practices across the internet. 
I suggest asking them to see how Out of the box Microsoft search works, and align with that.
This project can not accommodate their current requirement.
